docs/src/pages/css-in-js/basics/basics.md

```diff
@@ -46,8 +46,12 @@ import { install } from '@material-ui/styles';
 install();
 ```
 
-We will make `@material-ui/styles` the default style implementation for the core components in Material-UI v4.
-This installation step is temporary.
+It's **recommended** to place the above code in a separate file (e.g. `bootstrap.js`) and importing it in your application entry point (e.g. `index.js`).
+It ensures the installation is executed before anything else because ECMAScript imports are hoisted to the top of the module. If the installation step is not performed correctly the resulting build could have conflicting class names.
```

Comment:
```suggestion
This ensures that the installation is executed before anything else, because ECMAScript imports are hoisted to the top of the module. If the installation step is not performed correctly the resulting build could have conflicting class names.
```